spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -49,6 +49,9 @@ public class SpannerFixture : IAsyncLifetime, ICollectionFixture<SpannerFixture>
     public string FixedEncryptedDatabaseId { get; } = "fixed-enc-backup-db";
     public string FixedEncryptedBackupId { get; } = "fixed-enc-backup";
 
+    public string InstanceIdWithProcessingUnits { get; } = $"my-instance-processing-units-{DateTimeOffset.UtcNow.ToUnixTimeMilliseconds()}";
```

Comment:
On Dispose, I think you should attemp to delete this instance, inside a try catch so that we don't fail the tests if we can't delete it.

Reply:
Ah I was thinking about this and I got somewhat confused by where to do the delete because all other resources are deleted when they're "stale".

Added a method to delete the instance on dispose.